<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ShilpaTailor-PC" name="com.TestNG.NewTest" tests="2" failures="0" timestamp="13 Sep 2019 06:04:04 GMT" time="0.006" errors="1">
  <testcase name="testB" time="0.002" classname="com.TestNG.NewTest">
    <error type="org.testng.TestNGException" message="
Method public void com.TestNG.NewTest.testB(java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : data in class com.TestNG.Data_Providers">
      <![CDATA[org.testng.TestNGException: 
Method public void com.TestNG.NewTest.testB(java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : data in class com.TestNG.Data_Providers
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:263)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:419)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testB -->
  <testcase name="f3" time="0.004" classname="com.TestNG.NewTest"/>
</testsuite> <!-- com.TestNG.NewTest -->
